test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -5283,6 +5283,50 @@ BOOST_AUTO_TEST_CASE(simple_throw)
 	BOOST_CHECK(callContractFunction("f(uint256)", u256(1)) == encodeArgs());
 }
 
+BOOST_AUTO_TEST_CASE(strings_in_struct)
+{
+	char const* sourceCode = R"(
+		contract buggystruct {
+			Buggy public bug;
+			struct Buggy {
+				uint first;
+				uint second;
+				uint third;
+				string last;
+			}
+
+			function buggystruct(){
+				bug.first = 10;
+				bug.second = 20;
+				bug.third = 30;
+				bug.last = "asdfghjkl";
```

Comment:
Does this test actually trigger the bug? You do not call the struct constructer.